### ISSUE TITLE
Fix yml structure for sample files

### ISSUE THREAD

**Issue by mikarasv** [mikarasv/s-plode#4]
Use structure:
```yaml
---
prologue: |
symbolic-globals:
  - name: x
    type: x

main-set-up: | 
main-tear-down: | 
ansatz-call:
  name: x
  type: x
  arguments:
    - name: x
      symbolic: x
    - name: x
      symbolic: x
      type: x
```

Use yamale to verify structure

**Comment by mikarasv** [comment]
Closed on PR #15

**Comment by mikarasv** [comment]
Closed on PR #15 and #16
